src/TensorValues/Operations.jl

```diff
@@ -166,6 +166,19 @@ end
     Meta.parse("TensorValue{$D1,$D2}(($str))")
 end
 
+# a_ij = b_kij*c_k
+@generated function dot(a::A, b::B) where {A<:MultiValue{Tuple{D1,D2,D3}},B<:MultiValue{Tuple{D1}}} where {D1,D2,D3}
```

Comment:
This signature breaks our definition of single contraction. We always contract the last index of the first argument with the first index of the second argument (I.e. the indices that are in "contact"). We have two options:

Either move the c to the first argument
```
a = c ⋅ b #  a_ij = c_k*b_kij
```
or contract `j`
```
a = c ⋅ b #  a_kj = b_kij*c_j
```

Perhaps we want to include both.

Reply:
Sure. No problem. I will include both variations. In regards to the error in #414, do you have any ideas what might cause this?